A6lib.h

```diff
@@ -4,6 +4,7 @@
 #include <Arduino.h>
 #include "SoftwareSerial.h"
 
+#define DEBUG
```

Comment:
Setting `DEBUG` on in the library is only required if you're *actually* debugging the library, which most people aren't. Setting this always on will just increase the noise in their output, and may break things. `DEBUG` is usually specified in the command line of the specific build you're doing, or in your sketch (the .ino, yep).